about-us.html

```diff
@@ -19,10 +19,16 @@
 <body>
   <h1>Meet the Developers</h1>
   <div class="developers">
+    <div class="zoom-in">
     <img id="dev1" src="https://placehold.it/400x300/ddd" alt="Andrew Kyllo">
+    </div>
+    <div class="zoom-in">
     <img id="dev2" src="img/eugene-group.jpg" alt="Eugene Monnier">
+  </div>
+    <div class="zoom-in">
     <img id="dev3" src="https://placehold.it/400x300/ddd" alt="Robert Nielsen">
   </div>
+  </div>
   <div class="benefits">
     <h1>This game, is good for you!</h1>
```

Comment:
`<h1>This game, is good for you!</h1>`

Did you mean for the comma to be there or is it just a typo?